app/views/devise/sessions/new.html.haml

```diff
@@ -1,27 +1,27 @@
 - title "Login"
 .authform.devise
   - if resource.class.name == User.name
-    = content_tag(:span, nil, class: 'fa fa-key fa-3x', style: "display: block")
+    = content_tag(:span, nil, class: 'fa fa-key fa-3x', style: "display: block", "aria-hidden": "true")
 
   = form_for(resource, as: resource_name, url: session_path(resource_name), html: { role: 'form', class: 'form' }) do |f|
     %h2 Login
     .form-group
       = f.label :email
       %br
-      = f.email_field :email, autofocus: true, class: 'form-control'
+      = f.email_field :email, autofocus: true, class: 'form-control', autocomplete: 'email', required: true, "aria-required": "true"
```

Comment:
✨

Reply:
All this required are great, I wonder if we should add it as well to the signup and the reset password form since they throw an error when they are not set. 
Also maybe wont hurt to add an asterisk and a small label at the bottom "* Required fields" for people not using the screen reader. What do you think? probably that was more on the instructions criteria but maybe will be good to have.

Reply:
@anna-dorigo I added the required classes to the signup and reset password forms. I also added the "*Required fields" on forms that have more than one field, expect on the login page, which most people know require both an email and password. 👍 👏